include/stumpless/prival.h

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Copyright 2018-2022 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @file
+ * Prival codes for function to create severity and facility string from prival int value.  
+ */
+
+#ifndef __STUMPLESS_PRIVAL_H
+#  define __STUMPLESS_PRIVAL_H
+
+#  include <stumpless/config.h>
+#  include <stumpless/generator.h>
```

Comment:
use the header check tool (or the results of the Static Analysis CI check) to identify which include statements aren't needed - this one is unneeded at a minimum.